Habitica/src/main/java/com/habitrpg/android/habitica/HabiticaBaseApplication.java

```diff
@@ -201,17 +198,17 @@ private void setupFacebookSdk() {
     }
 
     private void setupCrashlytics() {
-        Crashlytics crashlytics = new Crashlytics.Builder()
+     /*   Crashlytics crashlytics = new Crashlytics.Builder()
```

Comment:
If this isn't used anymore, it should be commented out. If someone needs to reference it, it still exists in the git history.